android/src/com/mapswithme/maps/LocationState.java

```diff
@@ -6,11 +6,11 @@
 
   /// These values should correspond to values of
   /// location::EMyPositionMode defined in platform/location.hpp
-  public static final int UNKNOWN_POSITION = 0;
-  public static final int PENDING_POSITION = 1;
+  public static final int PENDING_POSITION = 0;
+  public static final int NOT_FOLLOW_NO_POSITION = 1;
```

Comment:
Not follow no position это разве не состояние?

> On 14.4.2016, at 12.32, Roman Kuznetsov <notifications@github.com> wrote:
> 
> In android/src/com/mapswithme/maps/LocationState.java:
> 
> > @@ -6,11 +6,11 @@
> >  
> >    /// These values should correspond to values of
> >    /// location::EMyPositionMode defined in platform/location.hpp
> > -  public static final int UNKNOWN_POSITION = 0;
> > -  public static final int PENDING_POSITION = 1;
> > +  public static final int PENDING_POSITION = 0;
> > +  public static final int NOT_FOLLOW_NO_POSITION = 1;
> Ну, логически это другое. UNKNOWN_POSITION - был состоянием, в которое переключаемся, с серой стрелочкой. Сейчас такого состояния нет.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
>